assets/queries/terraform/aws_bom/s3_bucket/query.rego

```diff
@@ -72,10 +72,28 @@ get_accessibility(bucket, bucketName) = accessibility {
 	accessibility = {"accessibility": "private", "policy": ""}
 } else = accessibility {
 	# cases when there is a unrestriced policy
-	acc := tf_lib.get_accessibility(bucket, bucketName, "aws_s3_bucket_policy", "bucket")
-    acc.accessibility == "hasPolicy"   
-    
-    accessibility = {"accessibility": "hasPolicy", "policy": acc.policy}   
+	acc := tf_lib.get_accessibility(bucket, bucketName, "aws_s3_bucket_policy", "bucket")  
```

Comment:
We should first verify ACL and then the policy. The policy should only be displayed when analyzed.

I suggest something like this:

```
get_resource_accessibility(resource, name) = info {
	is_public_access_blocked(resource.Properties.PublicAccessBlockConfiguration)
	info := {"accessibility": "private", "policy": ""}
} else = info {
	acl := get_bucket_acl(resource)
	lower(acl) == "private"
	info := {"accessibility": "private", "policy": ""}
} else = info {
	acl := get_bucket_acl(resource)
	lower(acl) != "private"
	info := {"accessibility": acl, "policy": ""}
} else = info {
    info := cf_lib.get_resource_accessibility(name, "AWS::S3::BucketPolicy", "Bucket")
} 
```